### PR TITLE
Implemented Series.argsort

### PR DESCRIPTION
**Pull request by itholic**
This PR proposes `Series.argsort`

```python
>>> kser = ks.Series([3, 3, 4, 1, 6, 2, 3, 7, 8, 7, 10])
>>> kser
0      3
1      3
2      4
3      1
4      6
5      2
6      3
7      7
8      8
9      7
10    10
dtype: int64

>>> kser.argsort().sort_index()
0      3
1      5
2      0
3      1
4      6
5      2
6      4
7      7
8      9
9      8
10    10
dtype: int64
```

### CODE REVIEW

**Comment by ueshin** [comment]
The result in the description is different from pandas'.

```py
>>> pd.Series([3, 3, 4, 1, 6, 2, 3, 7, 8, 7, 10]).argsort()
0      3
1      5
2      0
3      1
4      6
5      2
6      4
7      7
8      9
9      8
10    10
dtype: int64
```

**Comment by itholic** [comment]
> The result in the description is different from pandas'.
> 
> ```python
> >>> pd.Series([3, 3, 4, 1, 6, 2, 3, 7, 8, 7, 10]).argsort()
> 0      3
> 1      5
> 2      0
> 3      1
> 4      6
> 5      2
> 6      4
> 7      7
> 8      9
> 9      8
> 10    10
> dtype: int64
> ```

Oh, sorry I should've updated PR description.

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -5189,6 +5189,123 @@ def explode(self) -> "Series":
...
+        10    10
+        dtype: int64
+        """
+        sdf = self._kdf.to_spark()
+
+        seq_col_name = verify_temp_column_name(sdf, "__distributed_sequence_column__")
+        sdf = InternalFrame.attach_distributed_sequence_column(sdf, column_name=seq_col_name)
+
+        data_scol_name = self._internal.data_spark_column_names[0]
+        sdf = sdf.sort(data_scol_name).select(F.col(seq_col_name).alias(data_scol_name))
```
[the remark this answers is not in the document]
Comment (itholic) [reply databricks/koalas/series.py]: Thanks for the review and bug report :)

[the remark this answers is not in the document]
Comment (ueshin) [reply databricks/koalas/series.py]: The first one is not fixed yet.

[the remark this answers is not in the document]
Comment (itholic) [reply databricks/koalas/series.py]: I fixed it, but still running into some problem with

```python
>>> self = ks.Series([3, 3, 4, 1, 6, 2, 3, 7, 8, 7, 10], name='a.b')
>>> sdf = self._internal.spark_frame
>>> sdf.dropna()
Traceback (most recent call last):
...
pyspark.sql.utils.AnalysisException: Cannot resolve column name "a.b" among (__index_level_0__, a.b, __natural_order__); did you mean to quote the `a.b` column?;
```

I think maybe this is kinda bug from PySpark .. ??

[the remark this answers is not in the document]
Comment (ueshin) [reply databricks/koalas/series.py]: How about something like https://github.com/itholic/koalas/pull/3 ?

[the remark this answers is not in the document]
Comment (itholic) [reply databricks/koalas/series.py]: @ueshin Oh, thanks! I'll check it right away 😃

**Comment by itholic** [comment]
Thanks, @ueshin !

**Comment by itholic** [comment]
Anyway, I want to include @ueshin as an author this PR. 

Maybe merged pull request in my repository is enough, or I have to do something ??

**Comment by itholic** [comment]
Nvm about https://github.com/databricks/koalas/pull/1793#issuecomment-698657964 , think I'll be able to address it !

**Review by ueshin — APPROVED** [review]
LGTM.
Could you fix the code conflicts?

**Comment by itholic** [comment]
Oops. why the co-author didn't applied ? Sorry let me figure out the way.

**Pull request merged**